docs/SQL/development/target-models/sql-query.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SQL Query
+id: sql-query
+description: SQL Query of Target Models
+sidebar_position: 5
+tags:
+  - concept
+  - model
+  - query
+  - SQL
+---
+
+You can use the SQL Query tab to view and enable your custom SQL query at the end of your Target Model. A custom SQL query enables you to perform one last data transformation step as a SQL query, before creating the model.
+
+This is useful if you import your own codebase and your final query has additional conditions. Your query is saved here for you to view and edit. For low code users you can use a Filter Gem to achieve the same results.
+
+## Enable custom SQL query
+
+You can add a custom SQL Query if you're doing data processing with advance Jinja or dbt templating.
+This gives you flexibility when doing last mile operations for your SQL models. To that end, we support the use of declared variables directly in the SQL queries for those last mile operations on each model write.
+
+### Ports
+
+Ports represent Gem or generated code functions inputs and outputs. The port name is a variable name in Python and Scala code, and table alias in the SQL code. The visual order of the ports defines the order of the variables.
```

Comment:
I think this is not applicable in SQL.
`Ports represent Gem or generated code functions inputs and outputs. The port name is a variable name in Python and Scala code, and table alias in the SQL code.`

This part in it is still applicable, `table alias in the SQL code`. We can say Port represents the tables or gems that user wants to use in the SQL Query. They can access them using the aliases (port names).

Reply:
Done.